csrankings-c.csv

```diff
@@ -398,6 +398,7 @@ Chaofeng Sha,Fudan University,http://www.cs.fudan.edu.cn/?page_id=1918###,NOSCHO
 Chaokun Wang,Tsinghua University,http://ise.thss.tsinghua.edu.cn/~wangchaokun,NOSCHOLARPAGE
 Chaoli Wang 0001,University of Notre Dame,http://www3.nd.edu/~cwang11/about.html,z13bIdkAAAAJ
 Chaoning Zhang,Kyung Hee University,https://chaoningzhang.github.io,lvhxhyQAAAAJ
+Chaoqi Chen,Shenzhen University,https://github.com/chaoqichen,MmUZ_AQAAAAJ

```

Comment:
PhD student, not eligible for inclusion

Reply:
He is now an assistant professor at Shenzhen University. I have asked he change his google scholar info. Thanks.

Reply:
[Home page](https://chaoqichen.github.io/) also says he is a PhD student.

Reply:
Now is OK. I checked.

Thanks for your review.

Reply:
Now is OK. I checked.

Thanks for your review.

![image](https://github.com/user-attachments/assets/2bd6d78e-3b77-43b1-b9b6-ef51f482c28b)